src/pages/non-pass-finalisation/cat-be/non-pass-finalisation.cat-be.page.ts

```diff
@@ -152,6 +160,24 @@ export class NonPassFinalisationCatBEPage extends BasePageComponent implements O
         select(getTestSlotAttributes),
         select(isWelshTest),
       ),
+      transmission$: currentTest$.pipe(
+        select(getVehicleDetails),
+        select(getGearboxCategory),
+      ),
+      transmissionAutomaticRadioChecked$: currentTest$.pipe(
+        select(getVehicleDetails),
+        map(isAutomatic),
+        tap((val) => {
+          if (val) this.form.controls['transmissionCtrl'].setValue('Automatic');
```

Comment:
Will this auto-populate these values on page load? I think we changed this recently so the user had to explicitly input a value..

Reply:
This is how it is currently done on pass finalisation i believe.

Reply:
Ah this is for a fail so it doesn't matter :)

Reply:
ok :)